mcode/rems-admin
# Update encounter start and patient view cards

## Describe your changes

> Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

* Refactored code
* Included the absolute links seen on the (Turalio) Patient Enrollment card for the Patient Status Update card

Related rems-cds-hooks PR (pull down this repo's branch): https://github.com/mcode/rems-cds-hooks/pull/20

## Issue ticket number and Jira link

> Please include the Jira Ticket Number and Link for this issue/task.

[REMS-693](https://jira.mitre.org/browse/REMS-693)

[REMS-692](https://jira.mitre.org/browse/REMS-692)

## How to manually test
1. Use dev for all other repos except for rems-admin and rems-cds-hooks (inside src/cds-hooks) 
2. Clear out rems-admin and everything else in the settings tab in request-generator, also any local/session storage that affects the CDS hooks strings in the settings page
3. Select Jon Snow and Turalio
4. Click Send Rx
5. Click Sign Order
6. Fill out the Patient Enrollment Form
7. Go back to request-generator and select Jon Snow (to send the patient-view hook)
8. You should see a card with the Patient Status Update form with the dropdown containing informational links. This can also be verified in the Network tab's rems-patient-view network request response.
9. Go into the settings panel. Under the "Hook to send when selecting a patient" dropdown it is by default set to patient-view. Select encounter-start.
10. Redo steps 7-8 with the encounter-start hook.

## Checklist before requesting a review

## Checklist for conducting a review

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.
